sugargame/event.py

```diff
@@ -168,10 +169,10 @@ def _keyevent(self, widget, event, type):
             self.__keystate[keycode] = type == pygame.KEYDOWN
             if type == pygame.KEYUP:
                 mod = self._keymods()
-            ukey = unichr(Gdk.keyval_to_unicode(event.keyval))
+            ukey = chr(Gdk.keyval_to_unicode(event.keyval))
             if ukey == '\000':
                 ukey = ''
-            evt = pygame.event.Event(type, key=keycode, unicode=ukey, mod=mod)
+            evt = pygame.event.Event(type, key=keycode, str=ukey, mod=mod)
```

Comment:
Your change here, while important, was in the wrong commit.  Please confine changes into the commits.

Reply:
Will change it to include both of the change.